creation/web_base/cvmfs_helper_funcs.sh

```diff
@@ -503,3 +504,17 @@ perform_cvmfs_mount () {
 
         #loginfo "End log for mounting CVMFS"
 }
+
+glidein_config="$1"
+
+# import add_config_line function
+add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | cut -d ' ' -f 2-)
+# shellcheck source=./add_config_line.source
+. "$add_config_line_source"
+
+# adding system information about unprivileged user namespaces to the glidein classad
+glidein_work_dir=$(gconfig_get GLIDEIN_WORK_DIR "$glidein_config")
+unprivileged_user_namespaces=$(has_unpriv_userns)
+gconfig_add "HAS_UNPRIVILEGED_USER_NAMESPACES" "$unprivileged_user_namespaces"
+condor_vars_file=$(gconfig_get CONDOR_VARS_FILE "${glidein_config}" "-i")
```

Comment:
Looking into the [definition](https://github.com/namrathaurs/glideinwms/blob/8a7bee7aa1f6e1dce4081df7633b3926aa364584/creation/web_base/add_config_line.source#L257) for `add_condor_vars_line()` in `add_config_line.source`, the docstring says that the function uses the global variable `condor_vars_file`. If the global variable is not found, it throws a warning and returns an error.

Because of this, the existence of the `condor_vars_file` variable is necessary in order to use `add_condor_vars_line()` function

Reply:
Marco confirmed that we will proceed with the above.